README.md

```diff
@@ -59,15 +54,50 @@ implement that:
 
     response = http.request_get(uri.path, Authorization: access_token.authorization_header)
 
+## Service Provider Use Case
+
+    # Acquire Authorization header value from HTTP server's request
+    authz_header = request['Authorization']
+
+    # Parse the header value
+    access_token = AccessToken.parse_authorization_header(authz_header)
```

Comment:
Should this be `AccessToken. from_authorization_header` which uses `Protocol.parse_authorization_header` internally?

Reply:
Yeah, it should.

Reply:
Fixed at 26db3fb